src/main/kotlin/no/nav/omsorgspengerutbetaling/arbeidstakerutbetaling/JobbHosNåværendeArbeidsgiver.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.omsorgspengerutbetaling.arbeidstakerutbetaling
+
+import no.nav.helse.dusseldorf.ktor.core.ParameterType
+import no.nav.helse.dusseldorf.ktor.core.Violation
+
+data class JobbHosNåværendeArbeidsgiver(
+    val merEnn4Uker: Boolean,
+    val begrunnelse: Begrunnelse? = null
+) {
+    enum class Begrunnelse {
+        ANNET_ARBEIDSFORHOLD,
+        ANDRE_YTELSER,
+        LOVBESTEMT_FERIE_ELLER_ULØNNET_PERMISJON,
+        MILITÆRTJENESTE
+    }
+}
+
+fun JobbHosNåværendeArbeidsgiver.valider() = mutableSetOf<Violation>().apply {
+    if (merEnn4Uker == false && begrunnelse == null) {
```

Comment:
Var det slik at man skulle kreve vedlegg, dersom merEnn4Uker var true/false?
Isåfall må sjekkes at vedlegg ikke er tomt her.